components/elm/src/biogeophys/BareGroundFluxesMod.F90

```diff
@@ -301,8 +302,12 @@ subroutine BareGroundFluxes(bounds, num_nolakeurbanp, filter_nolakeurbanp, &
                um(p) = max(ur(p),0.1_r8)
             else                                      !unstable
                zeta = max(-100._r8,min(zeta,-0.01_r8))
-               wc = beta*(-grav*ustar(p)*thvstar*zii(c)/thv(c))**0.333_r8
-               um(p) = sqrt(ur(p)*ur(p) + wc*wc)
+               if ((.not. atm_gustiness) .or. force_land_gustiness) then
+                  wc = beta*(-grav*ustar(p)*thvstar*zii(c)/thv(c))**0.333_r8
+                  um(p) = sqrt(ur(p)*ur(p) + wc*wc)
+               else
+                  um(p) = max(ur(p),0.1_r8)
+               end if
```

Comment:
Should a check be added in during ELM's initialization to ensure that `atm_gustiness` and `force_land_gustiness` can not be both `.true.`?

Reply:
If by "offline" ELM you mean `I` cases or other cases without an active atmosphere, in those cases `atm_gustiness` is `.false.`, and so this code change has no effect (it takes the branch where `wc` is calculated).

The `force_land_gustiness` flag *only* has an effect if `atm_gustiness` is `.true.`. So if we do not allow them both to be `.true.`, `force_land_gustiness` should be removed entirely since it has no purpose.

Reply:
Thanks @quantheory for your explanation.